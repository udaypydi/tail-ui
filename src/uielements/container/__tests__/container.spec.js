import React from 'react';
import { render } from 'test-utils';
import  Container from 'uielements/container/Container.component.js';

it('should take a snapshot', () => {
    const { asFragment } = render(<Container />)
    
    expect(asFragment(<Container />)).toMatchSnapshot()
});

it('should render a raised container', () => {
    const { asFragment } = render(<Container className="w-full h-20 border border-gray-200" raised />)
    
    expect(asFragment(<Container className="w-full h-20 border border-gray-200" raised />)).toMatchSnapshot()
});

it('should render a fluid container', () => {
    const { asFragment } = render(<Container className="h-20 border border-gray-200" fluid />)
    
    expect(asFragment(<Container className="h-20 border border-gray-200" fluid />)).toMatchSnapshot()
});

it('should render a rounded container', () => {
    const { asFragment } = render(<Container className="h-20 border border-gray-200" rounded />)
    
    expect(asFragment(<Container className="h-20 border border-gray-200" rounded />)).toMatchSnapshot()
});

it('should render a rounded container', () => {
    const { asFragment } = render(<Container className="h-20 border border-gray-200 p-2" vertical>
    <p>Hello!!!</p>
    <p>Welcome to TailUI</p>
</Container>)
    
    expect(asFragment(<Container className="h-20 border border-gray-200 p-2" vertical>
    <p>Hello!!!</p>
    <p>Welcome to TailUI</p>
</Container>)).toMatchSnapshot()
});
