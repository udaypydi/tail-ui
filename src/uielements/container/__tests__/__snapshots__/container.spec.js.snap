// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a fluid container 1`] = `
<DocumentFragment>
  <div
    class="flex w-full flex-row h-20 border border-gray-200"
  />
</DocumentFragment>
`;

exports[`should render a raised container 1`] = `
<DocumentFragment>
  <div
    class="flex shadow-md flex-row w-full h-20 border border-gray-200"
  />
</DocumentFragment>
`;

exports[`should render a rounded container 1`] = `
<DocumentFragment>
  <div
    class="flex rounded flex-row h-20 border border-gray-200"
  />
</DocumentFragment>
`;

exports[`should render a rounded container 2`] = `
<DocumentFragment>
  <div
    class="flex flex-col h-20 border border-gray-200 p-2"
  >
    <p>
      Hello!!!
    </p>
    <p>
      Welcome to TailUI
    </p>
  </div>
</DocumentFragment>
`;

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-row"
  />
</DocumentFragment>
`;
